lib/queries.js

```diff
@@ -58,64 +86,82 @@ export async function createGoverningBody(administrativeUnit, classification) {
     ${PREFIXES}
 
     INSERT {
-        ${sparqlEscapeUri(uri)} a besluit:Bestuursorgaan ;
-          mu:uuid ${sparqlEscapeString(id)} ;
-          skos:prefLabel ?governingBodyLabel ;
-          besluit:bestuurt ${sparqlEscapeUri(administrativeUnit.uri)} ;
-          org:classification ${sparqlEscapeUri(classification)} .
+      ${sparqlEscapeUri(uri)} a besluit:Bestuursorgaan ;
+        mu:uuid ${sparqlEscapeString(id)} ;
+        skos:prefLabel ?governingBodyLabel ;
+        besluit:bestuurt ${sparqlEscapeUri(organization.uri)} ;
+        org:classification ${sparqlEscapeUri(classification)} .
     } WHERE {
-        ?administrativeUnit mu:uuid ${sparqlEscapeString(administrativeUnit.uuid)} ;
-          skos:prefLabel ?administrativeUnitLabel .
+      ?organization mu:uuid ${sparqlEscapeString(organization.uuid)} ;
+        skos:prefLabel ?organizationLabel .
 
-      BIND(CONCAT(${sparqlEscapeString(classificationLabel)}, " ", ?administrativeUnitLabel) as ?governingBodyLabel)
+      BIND(CONCAT(${sparqlEscapeString(classificationLabel)}, " ", ?organizationLabel) as ?governingBodyLabel)
     }
   `);
 
     return uri;
   } else {
-    throw Error(`classificationLabel not found ${classification}`);
+    throw Error(`classificationLabel not found for ${classification}`);
   }
 }
 
+/**
+ * Create a time specialisation of a given governing body.
+ * @param {string} governingBody - the URI of the governing body to specialise.
+ * @param {string} startDate - the starting date and time for the new governing
+ *     body.
+ * @param {string} [endDate] - the optional end data and time for the new
+ *     governing body.
+ * @returns {Promise<string>} the URI of the newly created time specialised
+ *     governing body.
+ */
 export async function createGoverningBodyInTime(
   governingBody,
   startDate,
-  endDate
+  endDate,
```

Comment:
***Suggestion***: Remove the extra comma. I guess is coming from your code formatter. Also line 128, 164